File: 프로그래머스/lv1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.java
class Solution {
    public long[] solution(int x, int n) {
	long[] answer = new long [n];
		double x3 =x;
		
		int x2 = x;

		for(int i=0 ; i<n;i++) {

			answer[i] = (long) x3;

			x3 = x3+ x2;

		}
		return answer;
    }
}

File: 프로그래머스/1/12954. x만큼 간격이 있는 n개의 숫자/x만큼 간격이 있는 n개의 숫자.js
function solution(x, n) {
    var answer = new Array(n);
    answer[0] = x ;
    var sum = x ; 
    for(var i = 1 ; i < n ; i++){
        answer[i] = answer[i-1] + sum ; 
    }
    
    return answer;
}